vignettes/relase-process.qmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "The Release Process"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{The Release Process}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+This document is adapted for The Hubverse from [The Carpentries Hubverse Developer's Handbook](https://carpentries.github.io/workbench-dev/releases.html) (c) The Carpentries. 
+
+## Checklist
+
+ - [ ] Create a new branch from `main` called `release-X.Y.Z`
+ - [ ] Update version number in DESCRIPTION and check that the `Remotes:` is
+       up-to-date
```

Comment:
What do you mean by `Remotes:` is up to date? Would it have been changed somewhere in the above steps described? If so where and why?

Reply:
This was a copy/paste artifact that describes an edge case (after development of cross-package functionality). 

Effectively, if you have two packages that have complementary features in two pull requests, you can test them in the pull request by changing the `Remotes:` field (this was before I fully understood the `Config/needs/*` section. 

```mermaid
flowchart LR
   user
   pkgA
   pkgB
   PRA1["pkgA@feature-A1"]
   PRB1["pkgB@feature-B1"]

   user-->|interface|pkgA
   pkgA-->|backend|pkgB
   PRB1-.->|pull request|pkgB
   PRA1-.->|pull request|pkgA
   PRA1-->|"Remotes:"|PRB1

```

Reply:
I've removed it as bumping the dev version number is sufficient to account for this.